mr_project/mr_app/views.py

```diff
@@ -1,9 +1,231 @@
+import json
+
 from django.shortcuts import render
 
 # Create your views here.
+from time import time
+from django.http import HttpResponse, HttpResponseBadRequest, HttpResponseNotFound, JsonResponse
+import neo4j
+from django.views.decorators.csrf import csrf_exempt
+from requests import codes
+from . import DRIVER, EMOTIONS, USERS
+from datetime import datetime
+
+#WRITE TRANSACTION FUNCTIONS
```

Comment:
Let's move any functions that handle neo4j transactions to a separate module

Reply:
I made a separate module, intelij won't recognize it at this point. I already agreed to add it to the local git repo.

Reply:
By module I mean python module (another file), not intellij module.

Reply:
I know, but I thought you could create a new python file using intelij, that is what I tried.